hudi-cli/src/main/java/org/apache/hudi/cli/commands/StatsCommand.java

```diff
@@ -73,14 +74,15 @@ public String writeAmplificationStats(
 
     List<Comparable[]> rows = new ArrayList<>();
     DecimalFormat df = new DecimalFormat("#.00");
+    TimelineLayout layout = TimelineLayout.getLayout(timeline.getTimelineLayoutVersion());
```

Comment:
getLayout -> fromVersionCode ?

Reply:
Prefer using fromVersion. Changed to the new name.